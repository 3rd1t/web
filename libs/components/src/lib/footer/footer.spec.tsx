import React from "react"
import renderer from "react-test-renderer"
import Footer from "./footer"

describe(" Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Footer bg="bg-carbon-900" primaryText="text-white" secondaryText="text-gray-200" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
