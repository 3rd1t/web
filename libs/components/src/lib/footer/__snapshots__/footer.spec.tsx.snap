// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Footer renders correctly 1`] = `
<footer
  className="bg-carbon-900"
>
  <div
    className="container mx-auto"
  >
    <div
      className="flex flex-wrap justify-between p-10  text-gray-200"
    >
      <div
        className="space-y-2"
      >
        <span
          className="text-xs font-semibold text-white uppercase"
        >
           Projects
        </span>
        <a
          className="block focus:outline-none hover:text-white"
          href="https://perfol.io"
        >
          Home
        </a>
        <a
          className="block focus:outline-none hover:text-white"
          href="https://analytics.perfol.io"
        >
          Analytics
        </a>
        <a
          className="block focus:outline-none hover:text-white"
          href="https://research.perfol.io"
        >
          Research
        </a>
      </div>
      <div
        className="space-y-2"
      >
        <span
          className="text-xs font-semibold text-white uppercase"
        >
           Resources
        </span>
        <a
          className="block focus:outline-none hover:text-white"
          href="/#"
        >
          Documentation
        </a>
        <a
          className="block focus:outline-none hover:text-white"
          href="/#"
        >
          Tutorials
        </a>
      </div>
      <div
        className="space-y-2"
      >
        <span
          className="text-xs font-semibold text-white uppercase"
        >
           Contact us
        </span>
        <a
          className="block focus:outline-none hover:text-white"
          href="mailto:info@perfol.io"
        >
          contact@perfol.io
        </a>
      </div>
      <div
        className="space-y-2"
      >
        <span
          className="text-xs font-semibold text-white uppercase"
        >
           Source
        </span>
        <a
          className="block focus:outline-none hover:text-white"
          href="https://github.com/perfolio"
        >
          github.com
        </a>
      </div>
    </div>
  </div>
  <div
    className="pt-2 border-t border-carbon-700"
  >
    <div
      className="max-w-6xl mx-auto"
    >
      <div
        className="flex flex-col items-center px-10 py-2 text-sm md:flex-row"
      >
        <span
          className="text-white"
        >
          Â© Copyright 
          2020
          . All Rights Reserved.
        </span>
        <div
          className="flex flex-row md:flex-auto md:flex-row-reverse"
        >
          <a
            className="block focus:outline-none hover:text-white"
            href="https://www.netlify.com"
          >
            <img
              alt="Deploys by Netlify"
              src="https://www.netlify.com/img/global/badges/netlify-light.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
